src/components/monitoringTable/MonitoringTable.tsx

```diff
@@ -0,0 +1,185 @@
+import {
+  Table,
+  TableRow,
+  Filters,
+  useFilters,
+  Pagination,
+  usePagination,
+  useAutocompleteTextInput,
+  FilterOption,
+} from '@pagopa/interop-fe-commons'
+import { Chip } from '@mui/material'
+import { ButtonNaked } from '@pagopa/mui-italia'
+import map from 'lodash/map'
+import parseInt from 'lodash/parseInt'
+import toLower from 'lodash/toLower'
+import toString from 'lodash/toString'
+import { useQuery } from '@tanstack/react-query'
+import apiRequests from '../../api/apiRequests'
+import { getServicesType, GetServicesResponseType } from '../../api/apiRequestTypes'
+import stores from '../../store/Store'
+import format from 'date-fns/format'
+import { useTranslation } from 'react-i18next'
+import { TableSkeleton } from '../skeleton/TableSkeleton'
+import { useNavigate } from 'react-router-dom'
+import { resolvePathVariables } from '../../utils/routerUtils'
+
+type EServiceListQueryFilters = {
+  eserviceName?: string
+  versionNumber?: string
+  producerName?: string
+  state?: string
+}
+
+export const MonitoringTable: React.FC = () => {
+  const { t } = useTranslation(['monitorTable', 'general'])
+  const [producersAutocompleteTextInput, setProducersAutocompleteTextInput] =
+    useAutocompleteTextInput()
+  const [updateSnackbar] = stores.useSnackbarStore((state) => [state.updateSnackbar])
+  const navigate = useNavigate()
+
+  async function fetchProducers(value: string): Promise<FilterOption[]> {
+    if (producersAutocompleteTextInput) {
+      let response = apiRequests.getProducers(value)
+      return response
+    }
+    return []
```

Comment:
Come ci siamo detti in call, si va a filtrare lato server dal terzo carattere in poi, quando invece è meno di tre caratteri si va a mostrare i primi risultati in db ordinati alfabeticamente (case-insensitive) per nome dell'e-service.

Lato FE quindi bisogna sempre fare la richiesta.
```suggestion
    return apiRequests.getProducers(value)
```


>NB: `useAutocompleteTextInput` implementa già la logica per cui se l'utente ha scritto meno di tre caratteri allora lo state che ritorna (in questo caso `producersAutocompleteTextInput`) è una stringa vuota, se invece l'utente scrive più di tre caratteri allora lo state è valorizzato con quei tre caratteri. [Qui](https://github.com/pagopa/interop-fe-commons/blob/develop/src/hooks/useAutocompleteTextInput.ts#LL24-L30C11) se volete dare un'occhiata all'implementazione


Da risolvere.